src/js/components/modal.js

```diff
@@ -12,21 +12,20 @@ const WRAPPER_CLASSNAME = `${MODAL_CLASSNAME}-wrapper`;
 const OPENER_ATTRIBUTE = "data-open-modal";
 const CLOSER_ATTRIBUTE = "data-close-modal";
 const FORCE_ACTION_ATTRIBUTE = "data-force-action";
+const NON_MODAL_HIDDEN_ATTRIBUTE = `data-modal-hidden`;
 const MODAL = `.${MODAL_CLASSNAME}`;
 const INITIAL_FOCUS = `.${WRAPPER_CLASSNAME} *[data-focus]`;
 const CLOSE_BUTTON = `${WRAPPER_CLASSNAME} *[${CLOSER_ATTRIBUTE}]`;
 const OPENERS = `*[${OPENER_ATTRIBUTE}][aria-controls]`;
 const CLOSERS = `${CLOSE_BUTTON}, .${OVERLAY_CLASSNAME}:not([${FORCE_ACTION_ATTRIBUTE}])`;
+const NON_MODALS = `body > *:not(.${WRAPPER_CLASSNAME}):not([aria-hidden])`;
```

Comment:
```suggestion
const NON_MODALS = `body > *:not(.${WRAPPER_CLASSNAME}):not([aria-hidden]):not(script)`;
```
Not that it matters, but for clarity we might want to exclude script tags too.

Reply:
I don't disagree with the premise, but I might worry it opens a can of worms as far as what other elements we might want to exclude (`style`, `link`, etc). Is there a particular scenario which would be improved by excluding them?

Reply:
Good point, we don't need it for functionality.